waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -353,16 +352,17 @@ proc startOnchainSync(g: OnchainGroupManager): Future[void] {.async.} =
 
   let ethRpc = g.ethRpc.get()
 
-  # the block chunk size decays exponentially with the number of blocks
-  # the minimum chunk size is 100
-  var blockChunkSize = 1_000_000
+  # static block chunk size
+  let blockChunkSize = 2_000
 
-  var fromBlock = if g.latestProcessedBlock.isSome():
+  var fromBlock = if g.latestProcessedBlock.isSome() and 
```

Comment:
similar comment regarding `latestProcessedBlock`. Why being an option? In this case every nwaku rln instance will have a latest processed block.

Have the feeling that you are treating `0` as `.isNone` and perhaps that can be simplified?

Reply:
in this case I think its a bit different. Isn't this treating 0 as `isNone`? When we can just remove the option?

Reply:
hmm here we're checking if it's safe to `.get()` the value from g.latestProcessedBlock

you're right, we can update it to not be an option here!

Reply:
Addressed in 7ca355e9!